docs/getting-started/dockstore-tools.rst

```diff
@@ -13,46 +13,39 @@ Tutorial Goals
 
 Assumptions
 -----------
-This document assumes you have already have a tool ready to register. You do not need to have a Dockerfile associated with it unless you are using one our legacy registration methods (more on that below).
+This document assumes you already have a tool ready to register. 
 
-If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. Of course, you can always follow along with your own unique tools too, which may or may not be associated with an autobuilding Docker image.
+If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. You can also follow this tutorial if your tool is not associated with an autobuilding Docker image.
 
-Regardless of how you made your tool, this tutorial will assume you are using CWL. For all other languages, :doc:`please see our documentation on workflows instead </getting-started/dockstore-workflows>`.
-
-.. note:: The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. However, Dockstore allows the registration of "WDL tools" using legacy registration methods. We encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>` instead.
+.. note:: Dockstore allows the registration of "WDL tools" using legacy registration methods. The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. We encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>` instead.
```

Comment:
This is a nitpick, but I feel that a qualifier is needed between the statements "you can register WDL tools" and "WDL doesn't have a concept of tools" to underline that the Dockstore and the WDL community have opposing definitions. If a qualifier is not given, then the contradiction between "you can register WDL tools" and "WDL doesn't have a concept of tools" is not being acknowledged, which may lead the user astray.

Basically I'm picturing Johnny User reading this and saying "You said you can register tools on Dockstore, then you say the WDL community says there is no difference between tools and workflows. That must mean either:
A) There is no meaningful difference between WDL tools and WDL workflows on Dockstore [false]
B) You made a typo and meant that the WDL community **does** make a distinction between them [false]"